src/controllers/tripsController.js

```diff
@@ -75,6 +78,7 @@ class tripsController {
         }
       });
     } catch (error) {
+      console.log(error);
```

Comment:
@RusimbiP it's better to remove ```console.log```  statements in the codebase.

Reply:
@hakoemmy What makes it better? What is your suggestion?

Reply:
@Patrick for me I think you should return the error you are catching instead of logging it

Reply:
@AbdoulNuru You're right